sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -195,9 +195,9 @@ $(document).on('click', '.postCourseNameChange', function (e) {
     url: url
     }).done(function( msg ) {
      $('.courseTitleTextContainer').hide();
-      reportSuccess('Course <strong>' + $thisCourseCode.text() + '</strong> renamed to <strong>' + msg.course_code) + '</strong>';
+      reportSuccess('Course <strong>' + $thisCourseCode.text() + '</strong> renamed to <strong>' + msg.course_code + '</strong>');
       $thisCourseCode.text(msg.course_code);
-      $thisCourseName.text(msg.name)
+      $thisCourseName.text(msg.name);
```

Comment:
Hi - it is. This is not an object being constructed - but several updates to the UI (changing the UI to whatever Canvas returns after a successful course name change.